functions.photon.php

```diff
@@ -271,3 +271,19 @@ function jetpack_photon_banned_domains( $skip, $image_url, $args, $scheme ) {
 
 	return $skip;
 }
+
+
+/**
+ * Jetpack Photon - Support Text Widgets.
+ * 
+ * @access public
+ * @param string $content Content from text widget.
+ * @return string
+ */
+function jetpack_photon_support_text_widgets( $content ) {
+    if ( class_exists( 'Jetpack_Photon' ) && Jetpack::is_module_active( 'photon' ) ) {
+        return Jetpack_Photon::filter_the_content( $content );
+    }
+
```

Comment:
This will work, but only if Photon is active. If the module is inactive, you're not returning anything in the Text widgets anymore, they will all be empty. You'd need to add a `return $content;` fallback here.